### PR TITLE
Add OAK-D demo

### PR DESCRIPTION
**Pull request by rafal-gorecki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,36 @@
-# depthai-docker
+# Docker Images for Luxonis OAX Series cameras
```
Comment (delihus) [inline README.md]:
```suggestion
# Docker Images for Luxonis OAK Series cameras
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
 
-Dockerized `depthai-ros` ROS package ([GitHub repo here](https://github.com/luxonis/depthai-ros))
+The repository includes a GitHub Actions workflow that automatically deploys built Docker images to the [husarion/depthai-docker](https://hub.docker.com/r/husarion/depthai) Docker Hub repositories. This process is based on [depthai-ros](https://github.com/luxonis/depthai-ros)repository.
```
Comment (delihus) [inline README.md]:
```suggestion
The repository includes a GitHub Actions workflow that automatically deploys built Docker images to the [husarion/depthai-docker](https://hub.docker.com/r/husarion/depthai) Docker Hub repository. This process is based on [depthai-ros](https://github.com/luxonis/depthai-ros) repository.
```

File: README.md [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
 
-1. Connect the camera with the USB cable.
+## Prepare Environment
 
-2. Clone this repository:
-```
-git clone https://github.com/husarion/depthai-docker.git
-cd depthai-docker/demo/oak-1
-```
+1. Plugin the Device
```
Comment (delihus) [inline README.md]:
```suggestion
1. Plug in the Device
```

File: README.md [hunk 4]
```diff
@@ -1,24 +1,36 @@
...
+You can use `lsusb` command to check if the device is visible.
 
-4. Launch it with the RViz:
-```
-xhost +local:docker
-docker compose up
-```
+## Demo
+
+1. Clone the Repository
```
Comment (delihus) [inline README.md]:
```suggestion
1. Clone the repository
```

File: README.md [hunk 5]
```diff
@@ -1,24 +1,36 @@
...
+
+   ```bash
+    git clone https://github.com/husarion/depthai-docker.git
+    cd depthai-docker/demo/oak-1
+   ```
+
+2. Activate the Device
```
Comment (delihus) [inline README.md]:
```suggestion
2. Activate the device
```

File: README.md [hunk 6]
```diff
@@ -1,24 +1,36 @@
...
+
+   ```bash
+   docker compose up luxonis
+   ```
+
+3. Launch Visualization
```
Comment (delihus) [inline README.md]:
```suggestion
3. Launch visualization
```

File: demo/oak-1-theora/Dockerfile [hunk 7]
```diff
@@ -1,4 +1,4 @@
 FROM ros:iron-ros-core
 
 RUN apt update && apt install -y \
-        ros-${ROS_DISTRO}-rqt*
\ No newline at end of file
+        ros-${ROS_DISTRO}-rqt*
```
Comment (delihus) [inline demo/oak-1-theora/Dockerfile]: I think it is not a good idea to install all `rqt` applications or plugins. Take a look:
https://index.ros.org/search/?term=rqt
image

Let's find what plugin is needed maybe only `rqt_image_view`?

**Review by delihus — CHANGES_REQUESTED** [review]

File: demo/oak-1-theora/Dockerfile (hunk 7 above)
Comment (rafal-gorecki) [inline demo/oak-1-theora/Dockerfile]:
```suggestion
        ros-${ROS_DISTRO}-rqt-image-view
```

**Review by delihus — APPROVED** [review]

**Pull request merged**